### PR TITLE
Improve PrestaShopBundle config structure

### PR DESCRIPTION
**Pull request by sarjon**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR aims to improve structure of PrestaShopBundle config
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | n/a
| How to test?  | Everything should work the same as it was before this PR.

When working on migration I noticed that it is kind of a mess in config/ directory.
Example: 
_admin/_  directory - has admin routing + services + application config  (navigation.yml).
_api/_  directory - again routing configuration, but for api only
_adapter/_  directory - adapter services only

This can still be improved by grouping services into related files.

Root config before:
before

Root config after:
after

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8885)

### CODE REVIEW

**Comment by sarjon** [comment]
what do you think @mickaelandrieu ?

**Comment by mickaelandrieu** [comment]
I think I wanted to do it since months :)

File: src/PrestaShopBundle/Resources/config/services/form_services.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+imports:
+    - { resource: form/form_type.yml }
+    - { resource: form/form_extension.yml }
```
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/config/services/form_services.yml]: misses an extra line at the end of file :)

**Review by mickaelandrieu — CHANGES_REQUESTED** [review]
only 1 comment then LGTM

**Comment by sarjon** [comment]
done @mickaelandrieu :)

File: src/PrestaShopBundle/Resources/config/services/form_services.yml (hunk 1 above)
Comment (sarjon) [reply src/PrestaShopBundle/Resources/config/services/form_services.yml]: done @mickaelandrieu

**Review by mickaelandrieu — APPROVED** [review]

**Comment by mickaelandrieu** [comment]
Thank you @sarjon,

ping @PrestaShop/prestashop-core-dev for information :)

**Comment by LittleBigDev** [comment]
:+1:

**Pull request merged**